pydata_sphinx_theme/static/webpack-macros.html

```diff
@@ -17,6 +17,7 @@
 
 {% macro head_pre_bootstrap() %}
   <link href="{{ pathto('_static/css/theme.css', 1) }}" rel="stylesheet" />
+<<<<<<< HEAD
```

Comment:
ah yeah - looks like you need to resolve this merge

Reply:
ah darn, thought the `yarn build` solved this. Cleared the merge conflicts, ran `yarn build` again...

Reply:
I have learned to always ignore merge conflicts in this auto-generated file, but just let the build regereate it ..